infra/modules/azure_cosmos_account/tests/apps/network_access/src/main.go

```diff
@@ -0,0 +1,101 @@
+package main
```

Comment:
question: why go? can't we use typescript instead? we publish our SDKs packages in TypeScript and we can use our existing GitHub Actions to run tests using a test suite like vitest

moreover this can be good code snippets for teams that want to use a client (and to instruct AI)

Reply:
Terratest is a go sdk

I think tests are for internal use only, I can't see how they can be used by application teams - we're testing modules here